tests/integration_tests/test_ingest_component_towns_fund.py

```diff
@@ -645,7 +645,7 @@ def test_ingest_with_r4_file_hs_file_failure(test_client, towns_fund_round_4_fil
         "title": "Bad Request",
         "validation_errors": [
             {
-                "cell_index": "DNone",
```

Comment:
This feels like a strange way to represent just a column without a row number, so have also changed this to strip the "None" and just say "D".